README.md

```diff
@@ -57,28 +59,106 @@ systemctl restart jans-auth
 
 ### Deployment
 
-Download the
-latest [agama-passkey.gama](https://github.com/GluuFederation/agama-passkey/releases/latest/download/agama-passkey.gama)
-file and deploy it in Auth Sever.
+Run these instructions on the server where you have `Janssen` or `Gluu` installed:
 
-Follow the steps below:
-
-- Copy (SCP/SFTP) the gama file of this project to a location in your `Jans Server`
-- Connect (SSH) to your `Jans Server` and open TUI: `python3 /opt/jans/jans-cli/jans_cli_tui.py`
-- Navigate to the `Agama` tab and then select `"Upload project"`. Choose the gama file
+- Download the
+  latest [agama-passkey.gama](https://github.com/GluuFederation/agama-passkey/releases/latest/download/agama-passkey.gama), you can use `wget`
+```
+wget https://github.com/GluuFederation/agama-passkey/releases/latest/download/agama-passkey.gama
+```
+- Open TUI: `python3 /opt/jans/jans-cli/jans_cli_tui.py`
```

Comment:
These are jans instructions with nothing specific to your project. Remove this whole section, or just reference the Jans docs about agama project deployment. If you have any ideas about how to make the official docs better for Agama project deployment, make a PR for Jans docs.

Reply:
You are right, so I just remove all those instructions and leave the .gif?
Or do I totally remove everything and place a reference link to a deployment of an agama project, like this https://docs.jans.io/v1.1.0/admin/developer/agama/quick-start-using-agama-lab/#deploy-agama-project?